launch.go

```diff
@@ -171,6 +195,24 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath string) error
 		return fmt.Errorf("fetching Pipeline ID %d: %v", j.PipelineID, err)
 	}
 
+	log.Printf("Fetching Parent Build %d for Meta", b.ParentBuildID)
```

Comment:
I see. I did not care about the first job. In the case of first job, `parentBuild Id` is nil or other value?